code/obj/machinery/computer/cloning.dm

```diff
@@ -248,6 +248,9 @@ ADMIN_INTERACT_PROCS(/obj/machinery/computer/cloning, proc/scan_someone, proc/cl
 	if (istype(subject.mutantrace, /datum/mutantrace/zombie))
 		show_message("Error: Incompatible cellular structure.", "danger")
 		return
+	if (istype(subject.mutantrace, /datum/mutantrace/vampiric_thrall))
```

Comment:
This is a pretty serious balance change, any reason why it's here?

Reply:
Vampire thrall mutant races are now correctly removed upon the death of the thrall, so the corpses of former thralls may still be scanned and cloned.
This was implemented as the cloner will re-add removed thrall mutantraces, ability holders, and abilities were the individual being cloned scanned while they were a thrall.

Reply:
Dying shouldn't remove the mutantrace though? Being cloned should remove it.